panels/templates/panel_applications/index.html

```diff
@@ -0,0 +1,246 @@
+{% extends "preregistration/preregbase.html" %}
+{% block title %}Panel Applications{% endblock %}
+{% block backlink %}{% endblock %}
+{% block content %}
+
+<script>
+    var showOrHideHeldBefore = function () {
```

Comment:
Hmm, that's probably a good idea, since I do hate having lots of functions like this which are so similar.  From thinking about what the function signature would look like though, this is probably the best I can come up with for how you'd call the utility method if it was fully generic:

```javascript
makeShowOrHider({
    event: 'click',
    source: 'held_before',
    target: 'past_attendance',
    condition: function (field) {
        return field.prop('checked');
    }
});
```

There are probably some ways we could make that shorter.  For example, if the function assumes that you're designating a checkbox, then we could instead just say

```javascript
makeShowOrHider({source: 'held_before', target: 'past_attendance'});
```

and then have some extra parameters we can use if it's a dropdown:

```javascript
makeShowOrHider({
    type: 'dropdown',
    source: 'presentation',
    target: 'other_presentation':
    show_if: {{ c.OTHER }}
});
```

How does that all look?  (In case it's not clear, you'd just call this at the ``<script></script>`` level and then you wouldn't need to define the function, or call the function and set the handler in the ``$(function () {  })`` block.)